vignettes/tibble.Rmd

```diff
@@ -97,9 +97,10 @@ To complement `tibble()`, tibble provides `as_tibble()` to coerce objects into t
 l <- replicate(26, sample(100), simplify = FALSE)
 names(l) <- letters
 
-timing <- microbenchmark::microbenchmark(
+timing <- bench::mark(
```

Comment:
There seems to be a negative interaction between the switch to bench and this line earlier in the vignette:

https://github.com/tidyverse/tibble/blame/434e725df62270c4ddeea1a80f4160cae812fa9c/vignettes/tibble.Rmd#L46

If I comment it out locally, `R CMD check` passes. What is that about?

Reply:
Could be an issue with _fansi_ or with how we print tibbles, https://github.com/brodieG/fansi/issues/54.

Reply:
Fyi, as a work-around to this problem you can use:

```
diff --git a/vignettes/tibble.Rmd b/vignettes/tibble.Rmd
index 84ba77b..b684d85 100644
--- a/vignettes/tibble.Rmd
+++ b/vignettes/tibble.Rmd
@@ -19,7 +19,8 @@ knitr::opts_chunk$set(collapse = TRUE, comment = pillar::style_subtle("#>"))
 
 colourise_chunk <- function(type) {
   function(x, options) {
-    lines <- strsplit(x, "\\n")[[1]]
+    # lines <- strsplit(x, "\\n")[[1]]
+    lines <- x
     if (type != "output") {
       lines <- crayon::red(lines)
     }
```
I am working on fixing `fansi` so it can handle this type of input, as it should, but the above work-around will mean you don't have to wait on my fix getting back on CRAN, which per CRAN rules should wait another couple of weeks (plus I need to test the fix carefully as it involves compiled code).